analyzers/src/SonarAnalyzer.CFG/SonarAnalyzer.CFG.cs.nuspec

```diff
@@ -7,6 +7,7 @@
     <authors>SonarSource</authors>
     <owners>SonarSource</owners>
     <license type="expression">LGPL-3.0-only</license>
+    <repository type="git" url="https://github.com/SonarSource/sonar-dotnet" />
```

Comment:
I've added it here for consistency. It's not really necessary at this point. 

Should we keep it?